src/components/NewDatePicker/index.js

```diff
@@ -88,7 +77,7 @@ class NewDatePicker extends React.Component {
      */
     // eslint-disable-next-line rulesdir/prefer-early-return
     handleOutsideClick(event) {
-        if (this.wrapperRef && !this.wrapperRef.contains(event.target) && this.wrapperRef !== event.target && this.state.isPickerVisible) {
+        if (this.wrapperRef.current && !this.wrapperRef.current.contains(event.target) && this.wrapperRef.current !== event.target && this.state.isPickerVisible) {
```

Comment:
```suggestion
        if (this.wrapperRef && this.wrapperRef.current && !this.wrapperRef.current.contains(event.target) && this.wrapperRef.current !== event.target && this.state.isPickerVisible) {
```
`this.wrapperRef` may not be set initially.

Note: `handleOutsideClick` may be removed once the other comments are resolved so treat this one with least priority.

Reply:
@ArekChr I believe this is resolved, please mark it as resolved.